### ISSUE TITLE
Th24 looks ugly when using a custom theme

### ISSUE THREAD

**Issue by dnchshrp** [Alex313031/thorium#863]
**System Details**
 - OS: Win10 21H2
 - Thorium Version [M128.0.6613.189 - 42nd Release](https://github.com/Alex313031/Thorium-Win/releases/tag/M128.0.6613.189)
M128

**Problem**
There are visible white lines that appear when using custom themes which makes the UI look ugly:
![image](https://github.com/user-attachments/assets/3414ab31-10b1-48c6-a81f-43c0e5b92fe8)
![image](https://github.com/user-attachments/assets/76d89ed2-adf0-408a-91fd-cc499ab3add2)
![image](https://github.com/user-attachments/assets/3336338d-5fc6-4916-94d2-5fe461e1c23d)
![image](https://github.com/user-attachments/assets/e3286168-9b23-4cd7-b59a-9940e15f36c4)

It also would be nice to be able to optionally make the tab separators invisible since they can look bad in some cases:
![image](https://github.com/user-attachments/assets/e964733a-040f-4f5d-84c7-9e79e245e809)

**Did you read the FAQ?**
Yes

**Additional Notes**
-

**Comment by Alex313031** [comment]
@gz83 @dnchshrp I can fix the white line outline on the omnibox. The white line under the bookmarks bar, im not sure what that is or why its doing that.

The tab separators are not an issue: they are behaving normally. Installing this theme on Chrome should also make the separators look like that. Removing the separators would fuck spacing up to no end, however, I might put a flag that tells them to be 0 opacity i.e. transparent to the color on the layer underneath it. The problem is that means yet another command line code path to maintain, just for you or the few that encounter this.

While I want everyone to enjoy thorium, and I am definitely not above taking the time to fix something just for one person, I also have to balance how many times I do stuff like that. I am thinking this should be easy, so I can probably do it for you, but no promises, because if it takes above a certain amount of time or requires modifying too many files, I won't be able to justify it in my head

**Comment by Alex313031** [comment]
@gz83 Yes, but the ultimate goal is to have Th24 work seamlessly, meaning that themes should also work. The only thing I have no idea about is the whole bookmarks bar white line thing.

**Comment by deviato** [comment]
I can confirm that when I activate th24 ui I have the same problem with that line, which is not white but transparent, what you see is the desktop below.

I am on Debian 12 Linux with XFCE.
